### PR TITLE
update develop config, add redacted guides config

### PR DESCRIPTION
**Pull request by donsizemore**
and roll in previous pull request i dropped the ball on merging.

### CODE REVIEW

File: guides.dataverse.org.xml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+#ssh -l user docs.dataverse.org &quot;(mkdir -p /var/www/docs.dataverse.org/en/$VERSION &amp;&amp; cd /var/www/docs.dataverse.org/en/$VERSION &amp;&amp; tar xvzf /tmp/$TARBALL)&quot;
+
+#(cd $WORKSPACE/doc/sphinx-guides; make clean; scl enable python27 &quot;make html SPHINXBUILD=/nfs/tools/lib/python/2.7/current/bin/sphinx-build&quot;; scl enable python27 &quot;make epub SPHINXBUILD=/nfs/tools/lib/python/2.7/current/bin/sphinx-build&quot;; cp build/epub/Dataverse.epub build/html; cd build/html; tar cvzf /tmp/$TARBALL .; scp /tmp/$TARBALL user@guides.dataverse.org:/tmp)
+
+#original build command with PDF
+#(cd $WORKSPACE/doc/sphinx-guides; make clean; make html; make latexpdf; cp build/latex/Dataverse.pdf build/html; make epub; cp build/epub/Dataverse.epub build/html; cd build/html; tar cvzf /tmp/$TARBALL .; scp /tmp/$TARBALL user@guides.dataverse.org:/tmp)
```
Comment (pdurbin) [inline guides.dataverse.org.xml:65]: 🎵 Memories... 🎵

File: guides.dataverse.org.xml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+# build without latexPDF which doesn&apos;t compile
+(cd $WORKSPACE/doc/sphinx-guides; make clean; make html; make epub; cp build/epub/Dataverse.epub build/html; cd build/html; tar cvzf /tmp/$TARBALL .; scp /tmp/$TARBALL user@guides.dataverse.org:/tmp)
```
Comment (pdurbin) [inline guides.dataverse.org.xml:68]: Is this scp to guides.dataverse.org real? Somebody went and created a user called "user" on that server?

**Review by pdurbin — COMMENTED** [review]
So glad to see the guides being built by Jenkins.

I wonder... does this handle pull requests too? Or could it someday?

**Comment by donsizemore** [comment]
> So glad to see the guides being built by Jenkins.
> 
> I wonder... does this handle pull requests too? Or could it someday?

the sphinx guides are currently part of the dataverse repo proper. we could break them off into their own repo, but I'd defer to @kcondon

**Comment by pdurbin** [comment]
Oh, no, sorry, I like that the docs are part of the main repo. I'm just thinking it would be good for people doing code review (doc review) to be able to see the HTML version of the docs. But this can be a manana thing, of course. Thanks!

**Comment by donsizemore** [comment]
> Oh, no, sorry, I like that the docs are part of the main repo. I'm just thinking it would be good for people doing code review (doc review) to be able to see the HTML version of the docs. But this can be a manana thing, of course. Thanks!

I think we can do this in Jenkins (the Git plugin calls this an "included region"). Let's do that in a separate issue.

**Pull request merged**